### ISSUE TITLE
[Feature request]: Please add the ability to customise line spacing when creating font

### ISSUE THREAD

**Issue by brzzdev** [MonoLisaFont/feedback#185]
### What feature are you proposing?

Any chance of adding a line spacing option to the customise page?

In Xcode, I have to set it to tight spacing to match up with SF Mono/Menlo/Deja Vu Sans/Cascadia Code/Julia Mono/Berkeley Mono's regular setting (my preferred spacing), but there are places where I can't change that, so I end up using an alternative :(

**Comment by bebraw** [comment]
For now, you could try https://github.com/source-foundry/font-line for adjustments.

**Comment by okonet** [comment]
Have you tried this setting?

<img width="942" alt="CleanShot 2023-01-20 at 09 16 22@2x" src="https://user-images.githubusercontent.com/11071/213648367-9c93864b-96d4-46c6-94f8-fb04fb2ff565.png">

<img width="206" alt="CleanShot 2023-01-20 at 09 17 27@2x" src="https://user-images.githubusercontent.com/11071/213648477-1579fdd1-a464-48bf-b580-9f028c19f008.png">

**Comment by brzzdev** [comment]
> In Xcode, I have to set it to tight spacing

Yeah, my issue is replicating that spacing in every other place I want to use a monospaced font

**Comment by okonet** [comment]
Most editors and terminals have similar settings. The problem is why we can’t change it is because of specific glyphs height that depend on that value. Thinks like power line symbols etc.
